src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -127,10 +127,11 @@ HTM_055=When no `viewBox` attribute is present, the values of the 'height' and '
 #media
 MED_001=Video poster must have core media image type.
 MED_002=%1$s element doesn't provide fallback.
-MED_003=Non-standard image resource of type %1$s found.
+MED_003=A manifest fallback must be provided for image resource '%1$s' of type '%2$s'.
 MED_004=Image file header may be corrupted.
 MED_005=Media Overlay audio reference %1$s to non-standard audio type %2$s found.
 MED_006=Some browsers do not support rendering SVG images which use a filename in the xlink:href property.
+MED_007=Within a 'picture' element, image sources must be Core Media Type Resources except for 'source' elements whose 'type' attribute is not a Core Media Type; found resource '%1$s' of foreign type '%2$s'.
```

Comment:
This is the only thing I'd maybe question. It's a mouthful to read this error. I wonder if we could shorten it to something like:

>  Foreign resources can only be referenced from a source element when its 'type' attribute is set.

Reply:
Yeah I didn't know how to turn this. I tried other wordings but they didn't work. I like yours better however, it's simpler!